client/src/store/subscriptions/onUserIsCurrentUserUpdate.js

```diff
@@ -0,0 +1,28 @@
+import { requests, meAPI } from "api";
```

Comment:
I think it would be good to have a simple test for this subscriber. Look at the `redux-mock-store` module for this. You should be able to setup a mock store, then dispatch a fake (synchronous) API_RESPONSE action with a response similar to what we get back from a users API endpoint. Depending on what's in that action, you'd expect to see the subscriber called and the state of the current user in the store to update.